src/bundler.ls

```diff
@@ -1,58 +1,24 @@
-require! <[ webpack path webpack-dev-server ]>
+require! <[ webpack path webpack-dev-server fs deep-extend ]>
+arch-webpack-config = require './webpack.config'
```

Comment:
Might be worth moving both `bundler` and `webpack.config` under a `bundler` or `build` folder.